source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,40 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
+  migration. The following events cannot occur: 
+  
+  - create
+  - drop
+  - dropDatabase
+  - createIndexes
+  - dropIndexes
+  - collModrefineCollectionShardKey
+  - rename
+  - reshardCollection
+  - shardCollection
```

Comment:
[question]

Should we monospace these?

Reply:
ah yes we should — overlooked that. updating!